components/MovieList/index.jsx

```diff
@@ -0,0 +1,38 @@
+import MovieCard from "../MovieCard";
+import Link from "next/link";
+import styled from "styled-components";
+import StyledLink from "../styledLink";
+
+const Ul = styled.ul`
+  list-style: none;
+`;
+
+const Li = styled.li`
+  margin-bottom: 1.6rem;
+`;
+
+const MovieList = ({ movies, movieInfo, onToggleFavorite }) => {
+  const movieResults = movies?.results || movies;
```

Comment:
instead of passing the "raw" data object that you receive from your API, you should change the code from your local api to simply pass the received data object as is to the frontend, but rather just send the array of movies from the "results" key back, e.g. 

```js
// in your api
const data = await response.json()
response.json(data.results)
```

Reply:
By this, you don't need to check if results exist, and movies is an array, as one would assume from the variable name.